t/chaos-test/kill-etcd_test.go

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package chaos
+
+import (
+	"net/http"
+	"os/exec"
+	"testing"
+
+	"github.com/gavv/httpexpect/v2"
+)
+
+var (
+	token = "edd1c9f034335f136f87ad84b625c8f1"
```

Comment:
Could we use a public file to store those constants?

Reply:
Maybe later when we have more test files, we could make them public? Since there are not too many constants here